src/braket/pennylane_plugin/translation.py

```diff
@@ -25,18 +26,82 @@
     StateVector,
     Variance,
 )
+from braket.schema_common import BraketSchemaBase
 from pennylane.operation import Observable, ObservableReturnTypes, Operation
 
 from braket.pennylane_plugin.ops import PSWAP, XY, YY, CPhaseShift00, CPhaseShift01, CPhaseShift10
 
+BRAKET_TO_PENNYLANE_OPERATIONS = {
```

Comment:
Make this dict private; it would be nice if we could somehow unify both translation directions, but that can be implemented under the hood later.